obspy/clients/eida/__init__.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+FDSN Web Service client with EIDA routing and authentication support.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+    Helmholtz-Zentrum Potsdam - Deutsches GeoForschungsZentrum GFZ
+    (geofon@gfz-potsdam.de)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+import io
+import obspy.clients.fdsn
+from . import fetch
+
+try:
+    # Python 2.x
+    import urlparse
+
+except ImportError:
+    # Python 3.x
+    import urllib.parse as urlparse
+
+class Client(obspy.clients.fdsn.Client):
+    def __init__(self, base_url='GFZ', retry_count=10, retry_wait=60, maxthreads=5, credentials=None, authdata=None, **kwargs):
+        """
+        Initializes an FDSN/EIDA Web Service client.
+
+        :type base_url: str
+        :param base_url: Base URL of FDSN/EIDA web service compatible server
+            (e.g. "http://geofon.gfz-potsdam.de") or key string for recognized
+            server.
```

Comment:
This should have a link to the respective page in the docs, i.e. ``.. see :mod:`FDSN client documentation <obspy.clients.fdsn>` ``